client/valor/schemas/symbolic/types.py

```diff
@@ -1493,6 +1512,9 @@ def __iter__(self) -> typing.Iterator[T]:
     def __len__(self) -> int:
         raise NotImplementedError
 
+    def contains(self, *other) -> Contains:
+        raise NotImplementedError
```

Comment:
so we're implementing `contains` for VariableList but not TypedList? why is that?